src/script/main/app.ts

```diff
@@ -372,6 +372,11 @@ export class App {
       onProgress(2.5);
       telemetry.timeStep(AppInitTimingsStep.RECEIVED_ACCESS_TOKEN);
 
+      const selfUser = await this.repository.user.getSelf([{position: 'App.initiateSelfUser', vendor: 'webapp'}]);
+
+      await initializeDataDog(this.config, selfUser.qualifiedId);
+      onProgress(5, t('initReceivedSelfUser', selfUser.name()));
```

Comment:
Nice little UX improvement, we show the user name in the loader earlier in the process :) 
No more empty loading bar for a while

Reply:
Was about to ask why this moved :D